Modules/xSharePoint/xSharePoint.psd1

```diff
@@ -24,7 +24,7 @@ Author = 'Microsoft Corporation'
 CompanyName = 'Microsoft Corporation'
 
 # Copyright statement for this module
-Copyright = '(c) 2015 Microsoft Corporation. All rights reserved.'
+Copyright = '(c) 2015 - 2016 Microsoft Corporation. All rights reserved.'
```

Comment:
Can I get you to roll this one back - I'll be doing a metadata update as part of the 1.0 release and wanna group those changes together there.